tcp-to-blob/deploy/deploy-in-docker.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+# Copyright (c) 2022 Microsoft Corporation. All rights reserved.
+# Software is licensed under the MIT License. See LICENSE in the project
+# root for license information.
+
+set -euo pipefail
+
+az account show # Fail fast if not logged in.
+PROJECT_DIR="$(dirname -- "$0")/../.."
+echo "PROJECT_DIR: \"${PROJECT_DIR}\""
+pushd "${PROJECT_DIR}"
+./install-node-modules.sh
+npx yarn build
+mkdir tcp-to-blob/.env
```

Comment:
```suggestion
mkdir -p tcp-to-blob/.env
```
If you re run the deploy, this errors out saying the dir already exists. Using -p will keep that error from popping up.